Add section on configuring anonymous login for OpenSearch Dashboards (#6739)
opensearch-project/documentation-website: _security/access-control/anonymous-authentication.md

If you disable anonymous authentication, you must provide at least one `authc` in order for the Security plugin to initialize successfully.
{: .important }

## OpenSearch Dashboards configuration

> To enable anonymous authentication for OpenSearch Dashboards, you need to modify the `opensearch_dashboards.yml` file inside the configuration directory of your OpenSearch Dashboards installation.

Reviewer: ```suggestion
To enable anonymous authentication for OpenSearch Dashboards, you need to modify the `opensearch_dashboards.yml` file in the configuration directory of your OpenSearch Dashboards installation.
```

Add the following setting to `opensearch_dashboards.yml`:

```yml
opensearch_security.auth.anonymous_auth_enabled: true
```

> Anonymous login for OpenSearch Dashboards requires that anonymous authentication is enabled on the OpenSearch cluster.

Reviewer: ```suggestion
Anonymous login for OpenSearch Dashboards requires anonymous authentication to be enabled on the OpenSearch cluster.
```
